kotlin/Deepgram-Topic-Detection/README.md

```diff
@@ -0,0 +1,63 @@
+# 🧹 Deepgram Transcribe Audio
+
+A Cloud Function for transcribing audio using [Deepgram](https://deepgram.com/)
+
+_Example input:_
+
+```json
+{
+    "fileUrl":"https://static.deepgram.com/examples/interview_speech-analytics.wav"
+}
+```
+
+_Example output:_
+
+
+```json
+{ 
+    "deepgramData": {}, 
+    "success": true 
+}
+```
+
+_Example error output:_
+
+
+```json
+{
+    "success":false,
+    "message":"Error"
+}
+```
+
+## 📝 Environment Variables
```

Comment:
Would you please remove references to Environment since those were in the older versions of runtimes:

```suggestion
## 📝 Variables
```